tests/utils/log/test_s3_task_handler.py

```diff
@@ -37,13 +39,14 @@ class TestS3TaskHandler(unittest.TestCase):
 
     def setUp(self):
         super(TestS3TaskHandler, self).setUp()
-        self.remote_log_location = 's3://bucket/remote/log/location'
-        self.remote_log_key = 'remote/log/location'
+        self.remote_log_base = 's3://bucket/remote/log/location'
+        self.remote_log_location = 's3://bucket/remote/log/location/1.log'
```

Comment:
This seems not to be required anymore?

Reply:
I think it sort of is, otherwise the new tests I added end up looking for the wrong file in th bucket